src/sortingtraining/Strategy.java

```diff
@@ -1,11 +1,26 @@
 package sortingtraining;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
 public enum Strategy {
+    COMPARE,
     BUBBLE,
     BUCKET,
-    EXIT,
-    ALL,
     INSERTION,
     DEFAULT,
-    QUICK
+    QUICK,
+    EXIT;
+
+    public static String list() {
+
+        List<String> str = new ArrayList<>();
+        Arrays.asList(Strategy.values()).forEach($ -> str.add($.toString()));
```

Comment:
Please use named variables. $ is not descriptive. One letter is more descriptive than that. Furthermore my IDE complains that variables should start from letter ;-)

You can write this as one liner:
you could return in in one line: 
```
        return Arrays.stream(Strategy.values())
                .map(Enum::toString)
                .collect(Collectors.joining(", "));
```

Reply:
That what i was looking for!! I knew that there is much quicker way!